README.md

```diff
@@ -1,87 +1,71 @@
 
-Come visit us at https://homeunite.us/
+The Home Unite Us team is working hard to create the technology that can meet those needs in a scalable fashion. Our pilot program allows guests and hosts to easily submit applications, and provides coordinators with a dashboard that makes the home matching process simpler. The stable version of Home Unite Us is at <https://homeunite.us/> and our most up to date beta features can be found at <https://dev.homeunite.us/>.
 
-### Project context
+Interested in joining our team? See our [contribution guide](CONTRIBUTING.md) to get started!
 
-This project is part of a larger initative at Hack for LA around creating a shared housing application, that can be used by organizations accross the county to help people tranisition to living independently and sustainably through host homes, empty bedrooms, and roomate matching for shared housing with lease signing.
+## Project Context
 
-## Technology used
+This project is part of a larger initiative at Hack for LA around creating a shared housing application, that can be used by organizations across the county to help people transition to living independently and sustainably through host homes, empty bedrooms, and roommate matching for shared housing with lease signing.
 
-### Frontend
-- [TypeScript](https://www.typescriptlang.org/)
-- [React](https://reactjs.org/docs/getting-started.html)
-- [Redux Toolkit](https://redux-toolkit.js.org/)
-- [Material UI](https://material-ui.com/)
-- [Vite](https://vitejs.dev/)
+## Technology Overview
 
-### Backend
-- [Python/Flask](https://flask.palletsprojects.com/en/1.1.x/)
-- [PostgreSQL](https://www.postgresql.org/)
-- [SQLAlchemy](https://www.sqlalchemy.org/)
-- [Docker](https://docs.docker.com/)
-- [Nginx](https://nginx.org/en/docs/)
-- [AWS](https://docs.aws.amazon.com/)
+The HomeUniteUs project is structured as a multi-[docker](https://docs.docker.com/) container application, with secret-protected access to networked resources. The project contains three containers, whose activities are coordinated using the `docker compose` configuration outlined in `docker-compose.yml`. The three containers are:
 
-# How to contribute
+1. `app`: A frontend [React](https://reactjs.org/docs/getting-started.html) app developed using [TypeScript](https://www.typescriptlang.org/).
+   * We use [Redux](https://redux.js.org/) to manage client state, with the [Redux Toolkit](https://redux-toolkit.js.org/) to simplify development.
+   * We use the [Material UI](https://material-ui.com/) component library, for access to high quality UI components designed with accessibility in mind.
+   * We use the [Vite](https://vitejs.dev/) build tool, for fast dev builds and optimized production builds.
+2. `api`: A backend python [Flask](https://flask.palletsprojects.com/en/1.1.x/) REST API, hosted on [AWS](https://docs.aws.amazon.com/).
```

Comment:
The HUU API is implemented using `connexion`. Let's say the API is a Python connexion application and as the bullet point, we can say we chose the variant that runs in top of Flask.

Reply:
Great clarification. So true. Corrected in [ebfba50](https://github.com/hackforla/HomeUniteUs/pull/557/commits/ebfba503f84ed549893cd2f68623dc90597fdafe)